src/lib/deployments/config/index.js

```diff
@@ -73,7 +73,38 @@ export function ingress() {
   return {
     ingress: {
       baseDomain,
-      class: `${releaseName}-nginx`
+      class: `${releaseName}-nginx`,
+      webserverAnnotations: {
+        "kubernetes.io/ingress.class": "{{ .Values.ingress.class }}",
+        "kubernetes.io/tls-acme": "{{ eq .Values.ingress.acme true | quote }}",
+        "nginx.ingress.kubernetes.io/custom-http-errors": "403,404,502,503",
+        "nginx.ingress.kubernetes.io/auth-url": "https://houston.{{ .Values.ingress.baseDomain }}/v1/authorization",
```

Comment:
```suggestion
        "nginx.ingress.kubernetes.io/auth-url": `${houston()}/v1/authorization`,
```

And 

```
import { houston, ui } from "utilities";
``` will be needed.